CITATION

```diff
@@ -1,153 +1,10 @@
 To cite the Dockstore project as a whole please refer to 
 
-O'Connor BD, Yuen D, Chung V et al. The Dockstore: enabling modular, community-focused sharing of Docker-based genomics tools and workflows [version 1; peer review: 2 approved]. F1000Research 2017, 6:52 (https://doi.org/10.12688/f1000research.10137.1) 
 
-To cite the Dockstore source code, please refer to:
+Denis Yuen, Louise Cabansay, Andrew Duncan, Gary Luu, Gregory Hogue, Charles Overbeck, Natalie Perez, Walt Shands, David Steinberg, Chaz Reid, Nneka Olunwa, Richard Hansen, Elizabeth Sheets, Ash O’Farrell, Kim Cullion, Brian D O’Connor, Benedict Paten, Lincoln Stein, The Dockstore: enhancing a community platform for sharing reproducible and accessible computational protocols, Nucleic Acids Research, Volume 49, Issue W1, 2 July 2021, Pages W624–W632, https://doi.org/10.1093/nar/gkab346
```

Comment:
There are newer team members, but I assume this refers to team members at the time of the paper, so it's OK?

Reply:
Yeah, it's generated from the NAR page for the paper